### PR TITLE
Refactor original code

### LINKED ISSUE #74: Refactor - Move existing method functionality into different py files
**What part of the code being refactored?**
The part of the code that is being refactored are all of the functionality of methods in the bot.py file.

**Why is the code being refactored?**
The code is being refactored because all the code is in one program which makes the file very large. It would also help to sort the methods into different files to easily find the method for each discord bot command.

**What are the changes to the code when being refactored?**
The changes are that are going to occur is that the functionality of the methods into different .py files which is sorted by the type of command of the discord bot.

### PR DESCRIPTION
**Pull request by klow358**
Refactored the code so each type of command are in separate .py files 
e.g. all the bot commands to do with watchlist management are in one python file

Closes #74

### CODE REVIEW

File: watchlist.json [hunk 1]
```diff
@@ -1 +1 @@
-{"watchlists": []}
\ No newline at end of file
+{"watchlists": [{"name": "t", "media": {"q": {"status": "unwatched", "tags": []}}, "participants": []}]}
```
Comment (Mman755) [inline watchlist.json]: Reset the watchlist.json just so it starts from a fresh state each time :)

```suggestion
{"watchlists": []}
```

**Review by Mman755 — APPROVED** [review]

**Comment by Mman755** [comment]
Refactoring looks good, we just might need to test it out a bit more since its a pretty big refactor. Good work though, just reset the watchlist.json and should be all goods.

File: commands/watchlist.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+def watchlist_delete_all():
+	watchlist_data = utils.read_watchlist_file(WATCHLISTFILENAME)
+	if len(watchlist_data["watchlists"]) == 0:
+		response = "There are no watchlists for me to delete ¯\_(ツ)_/¯"
+	else:
+		watchlist_data = {"watchlists": []}
+		utils.write_watchlist_file(WATCHLISTFILENAME, watchlist_data)
+		response = "Removed all watchlists, use /create to make a new one!"
+	return response
+def watchlist_view(watchlist_name):
```
Comment (tnai255) [inline commands/watchlist.py]:
```suggestion

# View a specified watchlist
def watchlist_view(watchlist_name):
```

File: watchlist.json [hunk 3]
```diff
@@ -1 +1 @@
-{"watchlists": []}
\ No newline at end of file
+{"watchlists": [{"name": "t", "media": {"q": {"status": "in progress", "tags": []}}, "participants": []}]}
```
Comment (tnai255) [inline watchlist.json]: Reset watchlist
```suggestion
{"watchlists": []}
```

**Review by tnai255 — CHANGES_REQUESTED** [review]
Overall lgtm. I think it would have still been good to have the method descriptors in the bot.py as well because if anyone were to open just bot.py they can still quickly know what it does. Then group them and on the top of the group have a comment like "All method logic are located in ____".

File: commands/media.py [hunk 4]
```diff
@@ -77,6 +75,15 @@ def watchlist_delete_media(media_name, watchlist_name):
 
 #Choose a media to watch at random from a watchlist
 def watchlist_choose(watchlist_name):
```
Comment (tnai255) [inline commands/media.py:77]: I personally think choose, clear, view watchlist should be in watchlist file cause they don't really need media info. But idk its okay if you think not!

**Review by tnai255 — APPROVED** [review]
LGTM, one suggestion but upto if you want to take on board or not :))

File: commands/media.py (hunk 4 above)
Comment (tnai255) [reply commands/media.py:77]: [Discussed in chat] - Decided to keep it as is since media information is being accessed.

**Pull request merged**